src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1438,7 +1438,7 @@ internal static void RemoveNestedModuleFunctions(PSModuleInfo module)
 #nullable enable
         private static void SortAndRemoveDuplicates<T>(List<T> input, Func<T, string> keyGetter)
         {
-            Dbg.Assert(input != null, "Caller should verify that input != null");
+            Dbg.Assert(input is not null, "Caller should verify that input != null");
```

Comment:
Please remove - we said about null _items_ in the List.

Reply:
I thought I would update the code style while touching the method.